ssv/pre_consensus_justification.go

```diff
@@ -9,23 +9,15 @@ import (
 
 // correctQBFTState returns true if QBFT controller state requires pre-consensus justification
 func (b *BaseRunner) correctQBFTState(msg *qbft.SignedMessage) bool {
-	inst := b.QBFTController.InstanceForHeight(b.QBFTController.Height)
-	decidedInstance := inst != nil && inst.State != nil && inst.State.Decided
-
-	// firstHeightNotDecided is true if height == 0 (special case) and did not start yet
-	firstHeightNotDecided := inst == nil && b.QBFTController.Height == msg.Message.Height && msg.Message.Height == qbft.FirstHeight
-
-	// notFirstHeightDecided returns true if height != 0, height decided and the message is for next height
-	notFirstHeightDecided := decidedInstance && msg.Message.Height > qbft.FirstHeight && b.QBFTController.Height+1 == msg.Message.Height
-
-	return firstHeightNotDecided || notFirstHeightDecided
+	// mind race conditions
+	return b.QBFTController.Height < msg.Message.Height
```

Comment:
1. no need for a separate func, just add it to shouldProcessJustifications
2. this means after the first valid pre-consensus justification msg, the next msg will not process pre-consensus justification. This means we need dedicated tests for it